### PR TITLE
Adding href link to external reference documentation

### PR DESCRIPTION
**Pull request by amalkurup89**
Description:

Adding href links to external documentation, so that user can navigate from Readme file

Motivation:

Document review

### CODE REVIEW

**Comment by amalkurup89** [comment]
@rogbas Can you please review and merge this change

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # splunk-forwarder-operator
 
-This operator manages Splunk Universal Forwarder. It deploys a daemonset which 
+This operator manages [Splunk Universal Forwarder](https://docs.splunk.com/Documentation/Forwarder/8.0.2/Forwarder/Abouttheuniversalforwarder). It deploys a daemonset which
```
Comment (wshearn) [inline README.md]: Can you link to https://docs.splunk.com/Documentation/Forwarder/latest/Forwarder/Abouttheuniversalforwarder instead?

Comment (amalkurup89) [reply README.md]: @wshearn Changed the link, as pointed

**Comment by wshearn** [comment]
Squish your comments please

**Comment by amalkurup89** [comment]
> Squish your comments please

@wshearn  Didn't get you. Do you want it as a single commit PR

**Comment by wshearn** [comment]
yeas, a single commit PR

**Comment by amalkurup89** [comment]
> yeas, a single commit PR

@wshearn Squashed the commits to a single one. Can you please approve and merge

**Comment by amalkurup89** [comment]
@wshearn Can you please approve

**Comment by amalkurup89** [comment]
@wshearn Can you please assign ok-to-test and lgtm tag to merge this change

**Comment by wshearn** [comment]
/ok-to-test
(someone else will have to lgtm)

**Comment by amalkurup89** [comment]
@wshearn One of the test is failing continuously. Should I retrigger test run again

**Comment by amalkurup89** [comment]
@wshearn Can you assign a reviewer, who can merge this change

**Comment by amalkurup89** [comment]
@wshearn can you assign lgtm tag or assign to someone, who can attach lgtm tag

**Pull request merged**